### PR TITLE
Features/project card design

### PR DESCRIPTION
**Pull request by nam-m**
## What
I updated the Projects page to match the current Figma design in [Wiki](https://github.com/OpenBCca/openbc-web/wiki/Project-Info#openbc-website-design). This PR contains the project card changes only, the Our Projects banner, and main header + footer layouts are not implemented yet.

### New project card details
- New fields such as "Languages", "Tools",.. (more in the pic below)
- GitHub link is added for each as an icon button
- New color-coded status label (top right corner of project card)
- Project cards length are adjusted to be equal to the longest one

Note: I added the Airbnb project and marked as 'Coming Soon'. The project pics are still fake.

## Figma Projects page
image

## Previous Projects page
image

## Current Projects page
image

### CODE REVIEW

File: src/app/projects/page.tsx [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
-                    <Typography>Sample Text</Typography>
-                  </CardContent>
-                  <CardActions>
-                    <Button size="small">View</Button>
-                    <Button size="small">Github</Button>
-                  </CardActions>
-                </Card>
+        <h1 className="projects-heading">Our Projects</h1>
+        <Container>
+          <Grid
```
Comment (PooriaT) [inline src/app/projects/page.tsx:18]: Isn't it the est practice to create a component for the part of the code between <container> tag?

File: src/components/project-overview/project-overview.tsx
Comment (PooriaT) [inline src/components/project-overview/project-overview.tsx:1]: I think this file has to be updated as it is behind the main's last commit.

**Comment by PooriaT** [comment]
Nice job, @nam-m 
The changes are smooth and great.

File: src/app/projects/page.tsx (hunk 1 above)
Comment (nam-m) [reply src/app/projects/page.tsx:18]: Agreed, i will make it a component

**Review by umsu2 — APPROVED** [review]

**Pull request merged**